src/options.rs

```diff
@@ -91,6 +91,20 @@ impl ResolvedRoundingOptions {
             .largest_unit
             .unwrap_or(smallest_unit.max(fallback_largest));
 
+        // 11. If LargerOfTwoTemporalUnits(largestUnit, smallestUnit) is not largestUnit, throw a RangeError exception.
+        // 12. Let maximum be MaximumTemporalDurationRoundingIncrement(smallestUnit).
+        // 13. If maximum is not unset, perform ? ValidateTemporalRoundingIncrement(roundingIncrement, maximum, false).
+        if largest_unit.max(smallest_unit) != largest_unit {
```

Comment:
Does `TemporalUnit` implement `Ord`? Because even if the spec says otherwise, I think it would be better to do a simpler `if largest_unit < smallest_unit`.

Reply:
Hmmmmmm, yeah that's probably true. I believe it should implement `Ord`